Mathlib/RepresentationTheory/GroupCohomology/LowDegree.lean

```diff
@@ -55,22 +58,22 @@ section Cochains
 
 /-- The 0th object in the complex of inhomogeneous cochains of `A : Rep k G` is isomorphic
 to `A` as a `k`-module. -/
-def zeroCochainsLinearEquiv : (inhomogeneousCochains A).X 0 ≃ₗ[k] A :=
+def zeroCochainsLequiv : (inhomogeneousCochains A).X 0 ≃ₗ[k] A :=
```

Comment:
I would prefer we keep the full `LinearEquiv` part in the names. If you would like to reduce the number of characters, you may alternatively use `cochains₀LinearEquiv`, etc.

Reply:
(Actually, there are some occurrences of `Lequiv` in the linear algebra files, so that it is presumably ok.)

Reply:
Yeah, I noticed `Lequiv` in some other files so just copied it.